docs/user/multi-user-dfuse.md

```diff
@@ -0,0 +1,114 @@
+# Multi user dfuse.
+
+## Overview
+
+Multi user dfuse is a operating mode of dfuse where it is able to provide filesystem access to
+multiple users/groups on a node through the same dfuse daemon.  This can be used persistently
+on a node to provide data sharing and scratch style access or on-demand by specific users to allow
+others to access their data.
+
+### Setup
+
+To enable multi user dfuse libfuse needs to be reconfigured as root to allow other users to access
+the mount point.  This is a per-node setting to a configuration file in /etc and will apply for all
+users on a node.  Only root can enable this, but once set it applies to all users.
```

Comment:
Should we go ahead and specify what the config setting is?

Reply:
Yes, this should probably be in the admin guide as well, I'll update.  There are security implications of this so we do not want to do it via rpm scripts for example and it will require a manual admin step.

Reply:
Admin guide is updated as part of this PR.